### ISSUE TITLE
Feature Request:

### ISSUE THREAD

**Issue by matamos** [ange-yaghi/engine-sim#90]
Feature Request: A page with sliders that change Stroke, Bore, Valves,and Cams  to Easily change performance Torque and Horsepower

**Comment by 14rss** [comment]
> Feature Request: A page with sliders that change Stroke, Bore, Valves,and Cams to Easily change performance Torque and Horsepower

Isn't this just making an engine? (ー_ーゞ

**Comment by HWiese1980** [comment]
Right. But currently you have to do these changes in code and restart the simulation every time. I'd love to have some sliders to adjust ignition timings, lobe shapes and stuff like that **at runtime**.

**Comment by sujaldev** [comment]
Maybe [this](https://github.com/Enginette/enginette)? Although it won't work at runtime.
